server/api.js

```diff
@@ -343,13 +343,16 @@ router.get("/avr/:channelId/:userId", async (req, res) => {
 
 router.get("/channelAvg/:channelId", (req, res) => {
 	const channelId = req.params.channelId;
+	console.log(channelId)
```

Comment:
Great! Your code is very clear to understand, I believe you need first to find the current day of week to calculate current week and then get last week and 2 last week .
Thank You Askin,

Reply:
Many thanks for reminding this Diba, I will do in my next branch.